### ISSUE TITLE
When I use streaming upload, streaming download, the memory will continue to increase, and finally crash

### ISSUE THREAD

**Issue by Coder-Miao** [ProxymanApp/atlantis#144]
![EA41D066-24FE-4909-8990-9BCCD9ECC98B](https://github.com/ProxymanApp/atlantis/assets/12209922/4b4da9c3-78ee-4081-bd27-c9212d497767)
<img width="1512" alt="EF9F6551-7B8B-4718-B1D4-0C66969B47A4" src="https://github.com/ProxymanApp/atlantis/assets/12209922/42296029-37f5-4f44-9955-ec0fa7b4381c">
- atlantis-proxyman (1.24.0)

This happens whether the client software is started or no！

**Comment by NghiaTranUIT** [comment]
Each request has ~1Mb Response body and it's a lot.

Does your app actively do it? I means: Make a lot of huge request.

**Comment by Coder-Miao** [comment]
Yes, the app is downloading concurrently. Uploading pictures will also have this situation!

**Comment by NghiaTranUIT** [comment]
thanks, so Atlantis captures a lot of huge requests -> Increase the RAM usage. It's a bug.

Atlantis should release RAM after sending the traffic to Proxyman macOS.

------
For now, I suggest using normal proxy. Here is the docs: https://docs.proxyman.io/debug-devices/ios-device
